src/ui/SmartSnippet/SmartSnippet.ts

```diff
@@ -202,7 +208,13 @@ export class SmartSnippet extends Component {
     if (lastRenderedResult) {
       this.renderSource(lastRenderedResult);
     }
-    this.heightLimiter.onContentHeightChanged();
+    this.shadowLoading.then(async () => {
+      if (!this.shadowContainer.attachShadow) {
```

Comment:
I'd prefer a `isIECompatibility` or something similar, extracted as a function name (so that it's more explicit that this is a workaround for IE11)

Reply:
Good point, I'll introduce `isShadowDomCompatible`.

Reply:
Edit: since `iframe` is becoming the default, this change isn't necessary anymore.